src/management/news_item.c

```diff
@@ -28,11 +28,42 @@
 #include "../world/sprite.h"
 #include "news_item.h"
 
-rct_news_item *newsItems = RCT2_ADDRESS(RCT2_ADDRESS_NEWS_ITEM_LIST, rct_news_item);
```

Comment:
For other lists I have now been declaring a global pointer which would eventually be converted to an array. In this case:

rct_news_item *gNewsItems = RCT2_ADDRESS(RCT2_ADDRESS_NEWS_ITEM_LIST, rct_news_item);

Then you can use gNewsItems[idx] in news_item_get().